### DISCUSSION TITLE
百姓苦房价久已

### DISCUSSION THREAD

**Discussion by SevenChen** [WeNeedHome/SummaryOfLoanSuspension#487] [General]
太苦了

**Comment by blacklion-spec** [comment]
08以后，超发货币导致，用高房价来减少市面上流通的货币

**Reply by dangoms** [reply]
《送君疫情歌》
疫情三年又三年
黄了恒大又融创
众人拍手又叫好
天道轮回看我朝
真命天子挽狂澜
资本收割靠边站
还民休养又生息
然而阵痛是事实
妖魔鬼怪趁机乱
各路神仙各边站
一场盛宴在眼前
软件码农裹其中
幻想投机把钱赚
且看世界今世道
资本收割老套路
皆谙其中危如卵
无人能抵其中利
不见棺材不落泪
维权路上看人性
不要皮脸骂社会
我劝君来有底线
俄乌都在飞蚱弹
你却想着大改善
又吃肉来又喝汤
只图自己舒坦过
可想出门见阎王
天不收人人自收
广厦千间三尺榻
劝君慈善把罪赎

**Comment by ghost** [comment]
當毛臘肉修宪廢除自由遷徙權利時候时候你說毛主席太偉大了
當毛臘肉51年殺150多w反革命時候，你為了一點點破地皮歡呼喝彩
當毛臘肉和周恩來狼狽為奸發動文革禍害知識分子時候，你當紅衛兵做幫兇
當鄧小平和江澤民在8964用坦克碾人民和學生的時候时候，你支持坦克
當習包子改宪法称帝时，你说我們習主席太伟大了
當香港為了返送中争取自己的自由時你支持香港警察
當BBC報道中共開閘導致洪水時候，你把人家趕出去了
當中共建立5 6億攝像頭監視公民有沒有反共時候，你在哪裡稱讚我們中國太安全了
當溫和派安倍，為中國災難籌錢，努力爭取中日 友好關係的和派安倍被刺殺時，你歡呼鼓舞，給中華土地蒙羞
當中共設置五個一政策阻止留學生回國時，你們罵他們千里投毒
當中國同胞跪在中共大使館門口哭著要回家時，你們說你們只提供聯繫家人的服務
當烏克蘭留學生因為戰亂求助大使館要回國時，你們坑人家18000的機票，還要半夜被叫起來歌唱祖國
當烏克蘭戰亂時，你們侮辱烏克蘭女性，讓在烏克蘭戰爭中受苦的中國同胞，只能被迫承認自己是日本人否則會被仇視，被屠殺

當你錢存銀行裡取不出來時候，我支持河南警察狠狠的打這幫刁民
當你房子烂尾时候，我支持中国用铁拳给我狠狠打這幫刁民
當得过COVID不允许进入场馆的时候，我支持中共警察給我狠狠抓這幫刁民
当发生唐山烧烤店打人的时候，我支持中共暴徒，給我狠狠打這幫刁民

你們自己奴性行為自己斷送了民主政府，自己迎接了獨裁集權政府，自己的所作所為讓所有幫助你的人都失望透頂。活該你現在這種狀態。希望小粉紅們可以記住你們的行為，否則你們就是下一個被坦克碾的群體

**Reply by blisee** [reply]
你别神经了，要不把证据提供出来，凭你说什么是什么啊

**Reply by remixzyb** [reply]
这些确实是事实发生的 我绝对没有太大问题

**Reply by 378804441** [reply]
台巴子就别出来丢人现眼了。先搞定你们菜马桶吧。一帮趴在大陆身上吸血的 寄生虫

**Reply by ppinkiris** [reply]
台巴子有时间关心关心岛内通胀都比复制粘贴有意义

**Reply by ppinkiris** [reply]
喝核废水吃莱猪确实会影响智力

**Reply by johnWick2017** [reply]
台巴子

**Reply by RedColoredElegy** [reply]
你们岛上的人终归是没有了解大陆的历史与现实，和那些粉红一样，不过是相信那些媒体的谎言罢了

**Comment by johnWick2017** [comment]
哈哈 我已经收到你的邮件，祝君好~
